python/tests/stress/arcticdb/version_store/test_stress_multicolumn.py

```diff
@@ -67,6 +69,10 @@ def test_stress_multicolumn(lib_type, request):
             [pd.DataFrame([dict_merge(dict(date=date), sd) for sd in sec_data]) for date in p], ignore_index=True
         )
 
+        if IS_PANDAS_TWO:
```

Comment:
An ideal normalisation should not require any special handling in unrelated tests. This is a symptom of breaking my first test.

Reply:
This has been removed in 5565ef1.